components-core/src/main/java/com/airwallex/android/core/Airwallex.kt

```diff
@@ -704,7 +704,45 @@ class Airwallex internal constructor(
             options,
             object : PaymentListener<PaymentIntent> {
                 override fun onSuccess(response: PaymentIntent) {
-                    listener.onCompleted(AirwallexPaymentStatus.Success(response.id))
+                    if (response.nextAction != null) {
+                        val provider = AirwallexPlugins.getProvider(ActionComponentProviderType.GOOGLEPAY)
+                        if (provider == null) {
```

Comment:
Is this really necessary given this method `confirmGooglePayPaymentIntent` can only be called when the component is available?